Index: epsoneplijs/Makefile.in
===================================================================
--- epsoneplijs.orig/Makefile.in	2013-11-21 20:13:50.000000000 +0700
+++ epsoneplijs/Makefile.in	2013-11-21 20:53:51.437973635 +0700
@@ -100,7 +100,8 @@
 	[ -d libieee1284 ] && (cd libieee1284 ; make clean) || true
 
 install: all
-	$(INSTALL) ijs_server_epsonepl$(EXE) -c $(bindir)/ijs_server_epsonepl$(EXE)
+	$(INSTALL) -d $(DESTDIR)/$(bindir)
+	$(INSTALL) ijs_server_epsonepl$(EXE) -c $(DESTDIR)/$(bindir)/ijs_server_epsonepl$(EXE)
 
 uninstall:
 	$(RM) $(bindir)/ijs_server_epsonepl$(EXE)
