diff --git a/Makefile.in b/Makefile.in
index 26b9de8..fc542ab 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -16,6 +16,7 @@ SHARED_LDFLAGS=-shared
 SHARED_OBJ=.so
 #SHARED_LDFLAGS=-dylib
 #SHARED_OBJ=.dylib
+LDFLAGS=
 
 FE=-o 
 IJS_EXEC_SERVER=ijs_exec_unix$(OBJ)
@@ -58,26 +59,26 @@ libijs.a:	$(LIB_OBJS)
 # makes sense to separate them, but they're small enough to make this probably
 # not worthwhile.
 libijs$(SHARED_OBJ):	$(LIB_OBJS)
-	$(CC) $(SHARED_LDFLAGS) $^ -o $@
+	$(CC) $(CFLAGS) $(LDFLAGS) $^ -o $@
 
 ijs_client_example$(EXE):	ijs_client_example$(OBJ) ijs_client$(OBJ) $(IJS_COMMON_OBJ) $(IJS_EXEC_SERVER)
-	$(CC) $(CFLAGS) $(FE)ijs_client_example$(EXE) ijs_client_example$(OBJ) ijs_client$(OBJ) $(IJS_COMMON_OBJ) $(IJS_EXEC_SERVER) 
+	$(CC) $(CFLAGS) $(LDFLAGS) $(FE)ijs_client_example$(EXE) ijs_client_example$(OBJ) ijs_client$(OBJ) $(IJS_COMMON_OBJ) $(IJS_EXEC_SERVER) 
 
 ijs_server_example$(EXE):	ijs_server_example$(OBJ) ijs_server$(OBJ) $(IJS_COMMON_OBJ)
-	$(CC) $(CFLAGS) $(FE)ijs_server_example$(EXE) ijs_server_example$(OBJ) ijs_server$(OBJ) $(IJS_COMMON_OBJ)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(FE)ijs_server_example$(EXE) ijs_server_example$(OBJ) ijs_server$(OBJ) $(IJS_COMMON_OBJ)
 
 ijs_server_epsonepl$(EXE): ijs_server_epsonepl$(OBJ) ijs_server$(OBJ) $(IJS_COMMON_OBJ) $(EPL_OBJS) $(EXTRA_DEPS) $(EPL_HEADERS)
-	$(CC) $(CFLAGS) $(FE)ijs_server_epsonepl$(EXE) ijs_server_epsonepl$(OBJ) ijs_server$(OBJ) $(EPL_OBJS) $(IJS_COMMON_OBJ) $(LDPATH) $(LDLIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(FE)ijs_server_epsonepl$(EXE) ijs_server_epsonepl$(OBJ) ijs_server$(OBJ) $(EPL_OBJS) $(IJS_COMMON_OBJ) $(LDPATH) $(LDLIBS)
 
 testlibusb$(OBJ): libusb/tests/testlibusb.c
 	$(CC) $(CFLAGS) -c -I./libusb -o testlibusb$(OBJ) libusb/tests/testlibusb.c
 
 testlibusb$(EXE): testlibusb$(OBJ) libusb/.libs/libusb.a
-	$(CC) $(CFLAGS) $(FE)testlibusb$(EXE) testlibusb$(OBJ) $(LDPATH) $(LDLIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(FE)testlibusb$(EXE) testlibusb$(OBJ) $(LDPATH) $(LDLIBS)
 
 #test5700lusb doesn't need a .o.c target as the source is in the same directory.
 test5700lusb$(EXE): test5700lusb$(OBJ) libusb/.libs/libusb.a
-	$(CC) $(CFLAGS) $(FE)test5700lusb$(EXE) test5700lusb$(OBJ) $(LDPATH) $(LDLIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(FE)test5700lusb$(EXE) test5700lusb$(OBJ) $(LDPATH) $(LDLIBS)
 
 # libusb.spec is a generated file that interfers with rpm building. Therefore removing as soon as possible 
 libusb/.libs/libusb.a:
@@ -99,7 +100,8 @@ clean: common_clean
 	[ -d libieee1284 ] && (cd libieee1284 ; make clean) || true
 
 install: all
-	$(INSTALL) ijs_server_epsonepl$(EXE) -c $(bindir)/ijs_server_epsonepl$(EXE)
+	$(INSTALL) -d $(DESTDIR)/$(bindir)
+	$(INSTALL) ijs_server_epsonepl$(EXE) -c $(DESTDIR)/$(bindir)/ijs_server_epsonepl$(EXE)
 
 uninstall:
 	$(RM) $(bindir)/ijs_server_epsonepl$(EXE)
