diff -p -up epsoneplijs-0.4.1/configure.in.old epsoneplijs-0.4.1/configure.in
--- epsoneplijs-0.4.1/configure.in.old	2008-09-05 14:34:39.000000000 -0400
+++ epsoneplijs-0.4.1/configure.in	2008-09-05 14:34:43.000000000 -0400
@@ -54,11 +54,11 @@ AC_ARG_WITH(libusb,[  --with-libusb     
   --without-libusb        disable libusb and full-feature USB support])
 
 AC_MSG_CHECKING(for libusb)
-if test "$with_libusb" != no -a -d libusb -a -f libusb/usb.h ; then
+if test "$with_libusb" != no -a -f /usr/include/usb.h ; then
 HAVE_LIBUSB="-DHAVE_LIBUSB -D_SVID_SOURCE"
-LIBUSB_EXTRA_LDPATH=-L./libusb/.libs
+LIBUSB_EXTRA_LDPATH=
 LIBUSB_EXTRA_LDLIBS=-lusb
-LIBUSB_EXTRA_DEPS=libusb/.libs/libusb.a 
+LIBUSB_EXTRA_DEPS=
 LIBUSB_EXTRA_OBJS=epl_utils_libusb.o
 LIBUSB_EXTRA_BINS=
 BID_EXTRA_OBJS=${BID_OBJS}
@@ -90,11 +90,11 @@ AC_ARG_WITH(libieee1284,[  --with-libiee
   --without-libieee1284   disable libieee1284 and full-feature parallel port support])
 
 AC_MSG_CHECKING(for libieee1284)
-if test "$with_libieee1284" != no -a -d libieee1284 -a -f libieee1284/include/ieee1284.h ; then
+if test "$with_libieee1284" != no -a -a -f /usr/include/ieee1284.h ; then
 HAVE_LIBIEEE1284="-DHAVE_LIBIEEE1284"
-LIBIEEE1284_EXTRA_LDPATH=-L./libieee1284/.libs
+LIBIEEE1284_EXTRA_LDPATH=
 LIBIEEE1284_EXTRA_LDLIBS=-lieee1284
-LIBIEEE1284_EXTRA_DEPS=libieee1284/.libs/libieee1284.a 
+LIBIEEE1284_EXTRA_DEPS=
 LIBIEEE1284_EXTRA_OBJS=epl_utils_libieee1284.o
 LIBIEEE1284_EXTRA_BINS= 
 BID_EXTRA_OBJS=${BID_OBJS}
diff -p -up epsoneplijs-0.4.1/epl_config.h.old epsoneplijs-0.4.1/epl_config.h
--- epsoneplijs-0.4.1/epl_config.h.old	2008-09-05 14:34:39.000000000 -0400
+++ epsoneplijs-0.4.1/epl_config.h	2008-09-05 14:34:44.000000000 -0400
@@ -123,11 +123,11 @@
 
 
 #ifdef HAVE_LIBUSB
-#include "libusb/usb.h"
+#include <usb.h>
 #endif 
 
 #ifdef HAVE_LIBIEEE1284
-#include "libieee1284/include/ieee1284.h"
+#include <ieee1284.h>
 #endif 
 
 #if defined(HAVE_LIBUSB) || defined(HAVE_KERNEL_USB_DEVICE) || defined(HAVE_LIBIEEE1284) || defined(HAVE_KERNEL_1284) || defined(HAVE_NULLTRANS)
diff -p -up epsoneplijs-0.4.1/epl_utils_libieee1284.c.old epsoneplijs-0.4.1/epl_utils_libieee1284.c
diff -p -up epsoneplijs-0.4.1/epl_utils_libusb.c.old epsoneplijs-0.4.1/epl_utils_libusb.c
--- epsoneplijs-0.4.1/epl_utils_libusb.c.old	2008-09-05 14:34:39.000000000 -0400
+++ epsoneplijs-0.4.1/epl_utils_libusb.c	2008-09-05 14:34:44.000000000 -0400
@@ -25,7 +25,7 @@
  * SOFTWARE.
 **/
 
-#include "libusb/usb.h"
+#include <usb.h>
 
 #include <stdlib.h>
 #include <string.h>
diff -p -up epsoneplijs-0.4.1/Makefile.in.old epsoneplijs-0.4.1/Makefile.in
--- epsoneplijs-0.4.1/Makefile.in.old	2008-09-05 14:34:39.000000000 -0400
+++ epsoneplijs-0.4.1/Makefile.in	2008-09-05 14:34:44.000000000 -0400
@@ -72,11 +72,11 @@ ijs_server_epsonepl$(EXE): ijs_server_ep
 testlibusb$(OBJ): libusb/tests/testlibusb.c
 	$(CC) $(CFLAGS) -c -I./libusb -o testlibusb$(OBJ) libusb/tests/testlibusb.c
 
-testlibusb$(EXE): testlibusb$(OBJ) libusb/.libs/libusb.a
+testlibusb$(EXE): testlibusb$(OBJ)
 	$(CC) $(CFLAGS) $(FE)testlibusb$(EXE) testlibusb$(OBJ) $(LDPATH) $(LDLIBS)
 
 #test5700lusb doesn't need a .o.c target as the source is in the same directory.
-test5700lusb$(EXE): test5700lusb$(OBJ) libusb/.libs/libusb.a
+test5700lusb$(EXE): test5700lusb$(OBJ)
 	$(CC) $(CFLAGS) $(FE)test5700lusb$(EXE) test5700lusb$(OBJ) $(LDPATH) $(LDLIBS)
 
 # libusb.spec is a generated file that interfers with rpm building. Therefore removing as soon as possible 
diff -p -up epsoneplijs-0.4.1/test5700lusb.c.old epsoneplijs-0.4.1/test5700lusb.c
--- epsoneplijs-0.4.1/test5700lusb.c.old	2008-09-05 14:34:39.000000000 -0400
+++ epsoneplijs-0.4.1/test5700lusb.c	2008-09-05 14:34:45.000000000 -0400
@@ -5,7 +5,9 @@
  */
 
 #include <stdio.h>
-#include "libusb/usb.h"
+#include <linux/limits.h>
+#include <sys/types.h>
+#include <usb.h>
 
 void print_endpoint(struct usb_endpoint_descriptor *endpoint);
 void print_altsetting(struct usb_interface_descriptor *interface);
