diff --git a/Makefile.in b/Makefile.in
index 26b9de8..149909d 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -16,6 +16,7 @@ SHARED_LDFLAGS=-shared
 SHARED_OBJ=.so
 #SHARED_LDFLAGS=-dylib
 #SHARED_OBJ=.dylib
+LDFLAGS=
 
 FE=-o 
 IJS_EXEC_SERVER=ijs_exec_unix$(OBJ)
@@ -58,26 +59,26 @@ libijs.a:	$(LIB_OBJS)
 # makes sense to separate them, but they're small enough to make this probably
 # not worthwhile.
 libijs$(SHARED_OBJ):	$(LIB_OBJS)
-	$(CC) $(SHARED_LDFLAGS) $^ -o $@
+	$(CC) $(CFLAGS) $(LDFLAGS) $^ -o $@
 
 ijs_client_example$(EXE):	ijs_client_example$(OBJ) ijs_client$(OBJ) $(IJS_COMMON_OBJ) $(IJS_EXEC_SERVER)
-	$(CC) $(CFLAGS) $(FE)ijs_client_example$(EXE) ijs_client_example$(OBJ) ijs_client$(OBJ) $(IJS_COMMON_OBJ) $(IJS_EXEC_SERVER) 
+	$(CC) $(CFLAGS) $(LDFLAGS) $(FE)ijs_client_example$(EXE) ijs_client_example$(OBJ) ijs_client$(OBJ) $(IJS_COMMON_OBJ) $(IJS_EXEC_SERVER) 
 
 ijs_server_example$(EXE):	ijs_server_example$(OBJ) ijs_server$(OBJ) $(IJS_COMMON_OBJ)
-	$(CC) $(CFLAGS) $(FE)ijs_server_example$(EXE) ijs_server_example$(OBJ) ijs_server$(OBJ) $(IJS_COMMON_OBJ)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(FE)ijs_server_example$(EXE) ijs_server_example$(OBJ) ijs_server$(OBJ) $(IJS_COMMON_OBJ)
 
 ijs_server_epsonepl$(EXE): ijs_server_epsonepl$(OBJ) ijs_server$(OBJ) $(IJS_COMMON_OBJ) $(EPL_OBJS) $(EXTRA_DEPS) $(EPL_HEADERS)
-	$(CC) $(CFLAGS) $(FE)ijs_server_epsonepl$(EXE) ijs_server_epsonepl$(OBJ) ijs_server$(OBJ) $(EPL_OBJS) $(IJS_COMMON_OBJ) $(LDPATH) $(LDLIBS)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(FE)ijs_server_epsonepl$(EXE) ijs_server_epsonepl$(OBJ) ijs_server$(OBJ) $(EPL_OBJS) $(IJS_COMMON_OBJ) $(LDPATH) $(LDLIBS)
 
 testlibusb$(OBJ): libusb/tests/testlibusb.c
 	$(CC) $(CFLAGS) -c -I./libusb -o testlibusb$(OBJ) libusb/tests/testlibusb.c
 
-testlibusb$(EXE): testlibusb$(OBJ) libusb/.libs/libusb.a
-	$(CC) $(CFLAGS) $(FE)testlibusb$(EXE) testlibusb$(OBJ) $(LDPATH) $(LDLIBS)
+testlibusb$(EXE): testlibusb$(OBJ)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(FE)testlibusb$(EXE) testlibusb$(OBJ) $(LDPATH) $(LDLIBS)
 
 #test5700lusb doesn't need a .o.c target as the source is in the same directory.
-test5700lusb$(EXE): test5700lusb$(OBJ) libusb/.libs/libusb.a
-	$(CC) $(CFLAGS) $(FE)test5700lusb$(EXE) test5700lusb$(OBJ) $(LDPATH) $(LDLIBS)
+test5700lusb$(EXE): test5700lusb$(OBJ)
+	$(CC) $(CFLAGS) $(LDFLAGS) $(FE)test5700lusb$(EXE) test5700lusb$(OBJ) $(LDPATH) $(LDLIBS)
 
 # libusb.spec is a generated file that interfers with rpm building. Therefore removing as soon as possible 
 libusb/.libs/libusb.a:
@@ -99,7 +100,8 @@ clean: common_clean
 	[ -d libieee1284 ] && (cd libieee1284 ; make clean) || true
 
 install: all
-	$(INSTALL) ijs_server_epsonepl$(EXE) -c $(bindir)/ijs_server_epsonepl$(EXE)
+	$(INSTALL) -d $(DESTDIR)/$(bindir)
+	$(INSTALL) ijs_server_epsonepl$(EXE) -c $(DESTDIR)/$(bindir)/ijs_server_epsonepl$(EXE)
 
 uninstall:
 	$(RM) $(bindir)/ijs_server_epsonepl$(EXE)
diff --git a/configure.in b/configure.in
index 2cb3dc3..bd59ceb 100644
--- a/configure.in
+++ b/configure.in
@@ -54,11 +54,11 @@ AC_ARG_WITH(libusb,[  --with-libusb           enable  libusb support (requires l
   --without-libusb        disable libusb and full-feature USB support])
 
 AC_MSG_CHECKING(for libusb)
-if test "$with_libusb" != no -a -d libusb -a -f libusb/usb.h ; then
+if test "$with_libusb" != no -a -f /usr/include/usb.h ; then
 HAVE_LIBUSB="-DHAVE_LIBUSB -D_SVID_SOURCE"
-LIBUSB_EXTRA_LDPATH=-L./libusb/.libs
+LIBUSB_EXTRA_LDPATH=
 LIBUSB_EXTRA_LDLIBS=-lusb
-LIBUSB_EXTRA_DEPS=libusb/.libs/libusb.a 
+LIBUSB_EXTRA_DEPS=
 LIBUSB_EXTRA_OBJS=epl_utils_libusb.o
 LIBUSB_EXTRA_BINS=
 BID_EXTRA_OBJS=${BID_OBJS}
@@ -90,11 +90,11 @@ AC_ARG_WITH(libieee1284,[  --with-libieee1284      enable  libieee1284 support (
   --without-libieee1284   disable libieee1284 and full-feature parallel port support])
 
 AC_MSG_CHECKING(for libieee1284)
-if test "$with_libieee1284" != no -a -d libieee1284 -a -f libieee1284/include/ieee1284.h ; then
+if test "$with_libieee1284" != no -a -a -f /usr/include/ieee1284.h ; then
 HAVE_LIBIEEE1284="-DHAVE_LIBIEEE1284"
-LIBIEEE1284_EXTRA_LDPATH=-L./libieee1284/.libs
+LIBIEEE1284_EXTRA_LDPATH=
 LIBIEEE1284_EXTRA_LDLIBS=-lieee1284
-LIBIEEE1284_EXTRA_DEPS=libieee1284/.libs/libieee1284.a 
+LIBIEEE1284_EXTRA_DEPS=
 LIBIEEE1284_EXTRA_OBJS=epl_utils_libieee1284.o
 LIBIEEE1284_EXTRA_BINS= 
 BID_EXTRA_OBJS=${BID_OBJS}
diff --git a/epl_config.h b/epl_config.h
index 6e2c2cb..f527924 100644
--- a/epl_config.h
+++ b/epl_config.h
@@ -123,11 +123,11 @@
 
 
 #ifdef HAVE_LIBUSB
-#include "libusb/usb.h"
+#include <usb.h>
 #endif 
 
 #ifdef HAVE_LIBIEEE1284
-#include "libieee1284/include/ieee1284.h"
+#include <ieee1284.h>
 #endif 
 
 #if defined(HAVE_LIBUSB) || defined(HAVE_KERNEL_USB_DEVICE) || defined(HAVE_LIBIEEE1284) || defined(HAVE_KERNEL_1284) || defined(HAVE_NULLTRANS)
diff --git a/epl_utils_libusb.c b/epl_utils_libusb.c
index ae93be7..7b391cd 100644
--- a/epl_utils_libusb.c
+++ b/epl_utils_libusb.c
@@ -25,7 +25,7 @@
  * SOFTWARE.
 **/
 
-#include "libusb/usb.h"
+#include <usb.h>
 
 #include <stdlib.h>
 #include <string.h>
diff --git a/test5700lusb.c b/test5700lusb.c
index 22184d5..f617218 100644
--- a/test5700lusb.c
+++ b/test5700lusb.c
@@ -5,7 +5,9 @@
  */
 
 #include <stdio.h>
-#include "libusb/usb.h"
+#include <linux/limits.h>
+#include <sys/types.h>
+#include <usb.h>
 
 void print_endpoint(struct usb_endpoint_descriptor *endpoint);
 void print_altsetting(struct usb_interface_descriptor *interface);
